modules/ebi-metagenomics/genomeproperties/tests/nextflow.config

```diff
@@ -0,0 +1,7 @@
+process{
+
+    withName: GENOMEPROPERTIES {
+        ext.args = '-all -outfiles table -outfiles web_json -outfiles summary'
```

Comment:
Why not having these in the main script?

Reply:
Because these are optional - in V5 we do output all of `-outfiles` parameters for example, but it could be we decide to only output one in the future so leaving that flexibility